metis-dereference/metis-dereference-import/src/main/java/eu/europeana/metis/dereference/vocimport/utils/DereferenceValidationUtils.java

```diff
@@ -0,0 +1,40 @@
+package eu.europeana.metis.dereference.vocimport.utils;
+
+import org.apache.commons.collections.CollectionUtils;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utils class focused on validating values used in dereferencing
+ */
+public class DereferenceValidationUtils {
```

Comment:
Isn't this a little over engineered? I mean it's one simple method that could simply be part of the VocabularyCollectionImporterFactory class instead.

Reply:
When I first created this class, it made sense to separate the "responsibilities".
But I agree with you, now it does look over engineered. I will fix this